frontend/src/components/CombatResults.vue

```diff
@@ -1,53 +1,67 @@
 <template>
   <div>
     <div class="tob-bg-img promotion-decoration">
-      <img class="vertical-decoration bottom" src="../assets/border-element.png">
+      <img class="vertical-decoration bottom" src="../assets/header-ads.png">
     </div>
-    <div class="float-left crypto-warrior-image">
+    <!-- <div class="float-left crypto-warrior-image">
       <img class="vertical-decoration" src="../assets/cryptowarrior.png">
-    </div>
+    </div> -->
```

Comment:
Clean up the code if it isn't required remove or comment a reason why it should stay